rainfall/theme.json

```diff
@@ -357,8 +357,10 @@
 		"elements": {
 			"button": {
 				"border": {
+					"color": "var(--wp--preset--color--black)",
 					"radius": "0",
-					"width": "0"
+					"style": "solid",
+					"width": "1px"
```

Comment:
there is a 1px difference still. set it to `2px`, to match the outline button border.

<img width="290" alt="image" src="https://user-images.githubusercontent.com/1935113/196708613-89c87bd8-e1ef-47cd-b940-7208ea5c4a0a.png">

Reply:
Thanks!! Done